integration_tests/integration/incidentDetailsSubmittedEdit.cy.ts

```diff
@@ -141,12 +143,14 @@ context('Incident details (edit after completion of report)', () => {
     incidentDetailsPage.reportingOfficerLabel().should('contain.text', 'Reporting officer')
     incidentDetailsPage.reportingOfficerName().should('contain.text', 'USER ONE')
   })
-  it('should show error if one of the time fields is not filled in correctly', () => {
+
+  it('should show error if one of the time fields is not filled in correctly - 3', () => {
     cy.visit(adjudicationUrls.incidentDetails.urls.submittedEdit('G6415GD', 34))
     const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
     incidentDetailsPage.timeInputHours().clear()
     incidentDetailsPage.timeInputMinutes().clear()
     incidentDetailsPage.timeInputHours().type('13')
+    incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
```

Comment:
I don't think this click is necessary? Yes is already selected I think as in the data, date of discovery is the same as incident date